### PR TITLE
Bushlack shopmatic

### PR DESCRIPTION
**Pull request by bushlack**
Removing obsolete shopmatic template and adding a new one for future integration with shopmatic's multiple entities.

### CODE REVIEW

File: myshopmatic.com.website.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {  
-   "providerId":"goshopmatic.hk",
+   "providerId":"goshopmatic.com",
```
Comment (arnoldblinn) [inline myshopmatic.com.website.json]: The file should be called goshopmatic.com.website.json with this providerId

File: myshopmatic.com.website.json [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
       {  
          "type":"CNAME",
          "host":"www",
-         "pointsTo":"%v2%",
+         "pointsTo":"%INPUTDOMAIN%",
          "ttl":"3600"
       }
    ]
-}
+}
```
Comment (arnoldblinn) [inline myshopmatic.com.website.json]: Where is the apex A record?  Don't they need this too?

File: myshopmatic.com.website.json (hunk 1 above)
Comment (bushlack) [reply myshopmatic.com.website.json]: Good catch - updated file name per suggestion.

File: myshopmatic.com.website.json (hunk 2 above)
Comment (bushlack) [reply myshopmatic.com.website.json]: I asked the provider about this.  They insisted they don't want one in the template for now.  They have 3 templates in GoDaddy's system that aren't in gitHub, and none of them have A records.

**Pull request merged**